### DISCUSSION TITLE
M Weather Station Project

### DISCUSSION THREAD

**Discussion by tonyis49** [jaydeethree/Home-Assistant-weatherdotcom#28] [General]
Hello,
Thank you for this super weather Integration.
I have successfully used it in my Weather Station Project using a Panlee 4" colour display panel. I had tried several other weather integrations but none worked as good or gave me the information I required quite as well as this one. Having all the extra sensors made it very easy to code into my openhasp.yaml 
Here are a couple of screenshots:

![20240313_095124](https://github.com/jaydeethree/Home-Assistant-weatherdotcom/assets/33941087/6dbc9c15-04b7-4bed-ae6e-b5acef40369d)
![20240313_095133](https://github.com/jaydeethree/Home-Assistant-weatherdotcom/assets/33941087/c8009298-37cc-4c65-9f8a-e48af42f6b27)

**Comment by jaydeethree** [comment]
Looks great, thanks for sharing this project!
